### ISSUE TITLE
Table crashes when combined with CharacterCount

### ISSUE THREAD

**Issue by mark-lumbao** [ueberdosis/tiptap#1550]
**Description**
The app crashes as soon as the set limit for character count is reached. But this behavior only happens when Table is used with CharacterCount.

**Steps to reproduce the bug**
1. Clone my test repo https://github.com/mark-lumbao/react-rich-text-editor
2. Install dependencies then run **npm run start:dev** 
3. Open the app in the browser
4. Insert a table then continue adding table content till the limit is reached

**Github Repo**
I created a Github repo to help you debug the issue:

* React: https://github.com/mark-lumbao/react-rich-text-editor

**Expected behavior**
As seen in the attached video, the app crashes when the limit is reached.

**Screenshot, video, or GIF**

https://user-images.githubusercontent.com/9671486/124768967-ace33c00-df6b-11eb-88c8-b26818ed3c69.mp4

**Environment?**
 - operating system: Linux Manjaro
 - browser: Brave Browser
 - mobile/desktop: Desktop
 - tiptap version: 2 beta

**Comment by yellow1912** [comment]
There is another issue with ChracterCount, the size of this inclusion is huge for whatever reason. I remember I saw a related issue but I cannot find it anywhere.

![image](https://user-images.githubusercontent.com/374134/125018770-f46ae480-e09f-11eb-8ee1-eb858214b76a.png)

**Comment by hanspagel** [comment]
@yellow1912 All packages include the core, the typical development setup (webpack, rollup …) will make sure to include the code only once. That’s called tree-shaking. You need to look at your bundle size, not the individual package size.
